packages/snap-store-mobx/src/types.ts

```diff
@@ -145,6 +145,7 @@ export type RecommendationStoreConfig = StoreConfig & {
 	batchId?: number;
 	settings?: {
 		variants?: VariantConfig;
+		bfCacheRestore: boolean;
```

Comment:
We have a similar setting for the `restorePosition` logic in the SearchController - it uses `restorePosition.onPageShow`. Perhaps we should use a similar name here? Maybe `searchOnPageShow` ?